samples/webview/webview.cpp

```diff
@@ -811,6 +814,16 @@ void WebFrame::OnTitleChanged(wxWebViewEvent& evt)
     wxLogMessage("%s", "Title changed; title='" + evt.GetString() + "'");
 }
 
+void WebFrame::OnContextMenu(wxContextMenuEvent& WXUNUSED(evt))
+{
+    //menu shown when default menu is disabled
```

Comment:
We might want to make this more obvious when running the sample too, e.g. by adding a dummy "This is a custom menu" item or something like that.